Modules/+Drivers/ArduinoServo.m

```diff
@@ -0,0 +1,59 @@
+classdef ArduinoServo < Modules.Driver
+    %ARDUINOSERVO Connects with a hwserver Arduino and uses one pin to control a servo.
+    %
+    % Call with the 1) hostname of the host computer (singleton based on host), and 2) the (integer) pin.
+    
+    properties (Constant)
+        hwname = 'Arduino';
+    end
+    properties (SetAccess=immutable)
+        connection
+        pin
+    end
+    properties (GetObservable, SetObservable)
+        angle = Prefs.Double(NaN, 'min', 0, 'max', 180, 'set', 'set_angle', 'allow_nan', true);
+    end
+    methods(Static)
+        function obj = instance(host, pin)
+            mlock;
+            persistent Objects
+            if isempty(Objects)
+                Objects = Drivers.ArduinoServo.empty(1,0);
+            end
+            [~,resolvedIP] = resolvehost(host);
+            for i = 1:length(Objects)
+                if isvalid(Objects(i)) && isequal({resolvedIP, pin}, Objects(i).singleton_id)
+                    obj = Objects(i);
+                    return
+                end
+            end
+            obj = Drivers.ArduinoServo(resolvedIP, pin);
+            obj.singleton_id = {resolvedIP, pin};
+            Objects(end+1) = obj;
+        end
+    end
+    methods(Access=private)
+        function obj = ArduinoServo(host, pin)
+            obj.connection = hwserver(host);
+            obj.com('?');
```

Comment:
Can a comment be added here explaining what this line does?

Reply:
No.

Reply:
Ok - blocking until it's added

Reply:
Maybe.